docs/reference/HailExpressionLanguage.md

```diff
@@ -433,7 +433,10 @@ Struct {
 In the above schema, `score` is the IMPUTE info score produced, and `nIncluded` is the number of samples with non-missing dosages.
 
 **Note:**
-If the genotype data was not imported using the [`importbgen`](commands.html#importbgen) or [`importgen`](commands.html#importgen) commands, then the results for all variants will be `score = null` and `nIncluded = 0`.
+If the genotype data was not imported using the [`importbgen`](commands.html#importbgen) or [`importgen`](commands.html#importgen) commands, then the results for all variants will be `score = NA` and `nIncluded = 0`.
+
+**Note:**
+It only makes sense to compute info score for an `Aggregable[Genotype]` per variant.  While performing a per-sample info score will run 
```

Comment:
This is incomplete... I assume you mean `While performing a per-sample info score will run, the result is not interpretable`